test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -2885,6 +2887,81 @@ public void EntryPointNaiveBayesMulticlass()
             TestEntryPointRoutine("iris.txt", "Trainers.NaiveBayesClassifier");
         }
 
+        [TensorFlowFact]
+        public void EntryPointImageClassification()
+        {
+            Env.ComponentCatalog.RegisterAssembly(typeof(ImageClassificationTrainer).Assembly);
+            Env.ComponentCatalog.RegisterAssembly(typeof(ImageAnalyticsEntryPoints).Assembly);
+
+            string dataFile = "images/images.tsv";
+            string dataDir = DataDir + "//images";
+
+            var dataPath = GetDataPath(dataFile);
+            var outputPath = DeleteOutputPath("model.zip");
+            
+            string inputGraph = string.Format(@"
+                {{
+                  'Nodes': [
+                    {{
+                      'Name': 'Data.CustomTextLoader',
+                      'Inputs': {{
+                        'InputFile': '$file1',
+                        'CustomSchema' : 'col=ImagePath:TX:0 col=Label:TX:0'
+                      }},
+                      'Outputs': {{
+                        'Data': '$data1'
+                      }}
+                    }},
+                    {{
+                      'Name': 'Transforms.Dictionarizer',
+                      'Inputs': {{
+                        'Column': [{{ 'Name': 'Label', 'Source': 'Label' }}],
+                        'Data': '$data1',
+                      }},
+                      'Outputs': {{
+                        'OutputData': '$data2'
+                      }}
+                    }},
+                    {{
+                      'Name': 'Transforms.ImageLoader',
+                      'Inputs': {{
+                        'Column': [{{ 'Name': 'Features', 'Source': 'ImagePath' }}],
+                        'Data': '$data2',
+                        'LoadAsBytes': true, 
+                        'ImageFolder': '{2}'
+                      }},
+                      'Outputs': {{
+                        'OutputData': '$data3'
+                      }}
+                    }},
+                    {{
+                      'Name': 'Trainers.ImageClassifier',
+                      'Inputs': {{
+                        'TrainingData': '$data3'
+                      }},
+                      'Outputs': {{
+                        'PredictorModel': '$model'
+                      }}
+                    }}
+                  ],
+                  'Inputs' : {{
+                    'file1' : '{0}'
+                  }},
+                  'Outputs' : {{
+                    'model' : '{1}'
+                  }}
+                }}", EscapePath(dataPath), EscapePath(outputPath), 
+            EscapePath(dataDir)
+            );
+
+            var jsonPath = DeleteOutputPath("graph.json");
+            File.WriteAllLines(jsonPath, new[] { inputGraph });
+
+            var args = new ExecuteGraphCommand.Arguments() { GraphPath = jsonPath };
+            var cmd = new ExecuteGraphCommand(Env, args);
+            cmd.Run();
+        }
```

Comment:
this is basically copy-paste of TestEntryPointRoutine takes list of transforms and a trainer to test. Have you considered using TestEntryPointRoutine and passing list of transforms and a trainer to it?